jdaviz/configs/imviz/plugins/catalogs/catalogs.vue

```diff
@@ -28,6 +28,24 @@
       <g-file-import id="file-uploader"></g-file-import>
     </plugin-file-import-select>
 
+    <v-row v-if="catalog_selected === 'Gaia'">
+      <j-docs-link>
+        See the <j-external-link link='https://astroquery.readthedocs.io/en/latest/gaia/gaia.html' linktext='astropy.gaia docs'></j-external-link> for details on other configurable features.
```

Comment:
In the same spirit as Kyle's comment, we could make it clear that we're pointing them to the astroquery docs so they can see what defaults we're sort of forcing them to use in the plugin:
```suggestion
        See the <j-external-link link='https://astroquery.readthedocs.io/en/latest/gaia/gaia.html' linktext='astropy.gaia docs'></j-external-link> for details on the query defaults.
```